spec/fixtures/hieradata/rsyslog_tls_with_pp.yaml

```diff
@@ -0,0 +1,4 @@
+---
+rsyslog::enable_tls_logging: true
+simp_rsyslog::forward::permitted_peers: "*.my.domain,this.system"
+simp_rsyslog::forward::enable_tls: true
```

Comment:
This parameter does not exist.  Looks like cruft from an earlier iteration of this PR.